core/mocks/flags.go

```diff
@@ -0,0 +1,1060 @@
+// Code generated by mockery v2.14.0. DO NOT EDIT.
```

Comment:
oh wait these are already generated in core/internal/mocks? Via the go_generate I put in there

Reply:
ah I see what happened - you should remove the mockery calls from gethwrappers/go_generate.go, rm -rf core/mocks/ then and then all will be well